### PR TITLE
[Package Add] Clib

### PR DESCRIPTION
**Pull request by D-Brox**
Adds `clib`, the  C package manager-ish

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Also edit `packagelist`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/clib/clib.pacscript [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+depends="gcc libcurl4-gnutls-dev"
+hash="99640506e1cf69d69451dc5d0db2d8fd169ff8247bda8c6d01dd66e9471ac6a0"
+pkgdir="/usr/src/pacstall/${name}"
+removescript="yes"
+
+prepare () {
+  true
+}
+build() {
+  sudo make
```
Comment (wizard-28) [inline packages/clib/clib.pacscript:14]: Why `sudo make`?

Comment (D-Brox) [reply packages/clib/clib.pacscript:14]: My mistake, only `make install` needs sudo

File: packages/clib/clib.pacscript [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+depends="gcc libcurl4-gnutls-dev"
+hash="99640506e1cf69d69451dc5d0db2d8fd169ff8247bda8c6d01dd66e9471ac6a0"
+pkgdir="/usr/src/pacstall/${name}"
+removescript="yes"
+
+prepare () {
+  true
+}
+build() {
+  make
```
Comment (wizard-28) [inline packages/clib/clib.pacscript:14]:
```suggestion
  make -j"${nproc}"
```
Use all cores

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**